.gitmodules

```diff
@@ -1,23 +1,23 @@
 [submodule "MCServer/Plugins/Core"]
-	path = MCServer/Plugins/Core
+	path = Server/Plugins/Core
 	url = https://github.com/mc-server/Core.git
 [submodule "MCServer/Plugins/ProtectionAreas"]
```

Comment:
Yes, that would work, but everyone would have to execute that in their repo everytime they checked out something across this commit.